common/enclave/cc_data.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include "pdo/common/crypto/crypto.h"
+#include "pdo/common/types.h"
+
+class cc_data
```

Comment:
I would suggest to call this class `cc_state` instead of `cc_data` to better align with the diagrams where we talk about `sealed state` as well. Also `interfaces.md` uses the term state.